app/templates/rules.html

```diff
@@ -84,7 +84,12 @@ <h3 class="panel-title">
     <div class="left" style="float: left">
 
       <h5 title="Mapping">
-        Mapping: <b ng-bind-html="highlightSearch(rule.mapping, 'mapping')"></b>
+        Mapping: <span ng-show="rule.mapping">
+            <b ng-bind-html="highlightSearch(rule.mapping, 'mapping')"></b>
+            <!--<a href="/releases/#{{ rule.mapping }}"><b ng-bind-html="highlightSearch(rule.mapping, 'mapping')"></b></a>-->
+            <a href="" ng-show="rule.mapping" ng-click="openReleaseDataModal(rule.mapping)" class="glyphicon glyphicon-eye-open" title="View"></a>
+            <a href="/releases/#{{ rule.mapping }}" class="glyphicon glyphicon-pencil" title="Edit"></a>
```

Comment:
I dug into this a bit more and it looks like the nav links use <a href> (https://github.com/mozilla/balrog-ui/blob/ac8b624d4095ecdefd59cf5fe8b55f5397d5cec8/app/pages/index.us#L24). Given that, I'm going to stick with that here, and hopefully it can get improved when when we upgrade to the latest version of Angular.